projects/cm_mcu/cm_mcu.c

```diff
@@ -254,19 +255,14 @@ int main(void)
 
   // mutex for the UART output
   xUARTMutex = xSemaphoreCreateMutex();
```

Comment:
please move this also to the new Semaphore.{c,h} files

Reply:
Done.